.claude/commands/build.md

```
@@ -227,6 +238,26 @@ Using autoIncrement in eas.json handles build numbers automatically.
 - [ ] App exists in App Store Connect
 
+**iOS ASC Key File Check (iOS only — skip if platform is Android):**
+
+Read `eas.json` to extract `ascApiKeyPath` from the iOS submit profile, then verify the file exists:
+
+```bash
+ls {ascApiKeyPath} 2>&1
+```
+
+- If missing → 🚫 BLOCKING:
+  ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)